### ISSUE TITLE
PyXB doc link on pypi is out of date

### ISSUE THREAD

**Issue by nirgilboa** [pabigot/pyxb#90]
The documentation link on pypi page (https://pypi.python.org/pypi/PyXB) refers to http://pythonhosted.org/PyXB/ (version at 1.2.2)
Whereas the latest docs seem to be located at: https://pabigot.github.io/pyxb/ (version at 1.2.6)

**Comment by pabigot** [comment]
Yes; since the previous release PyPI disabled the ability to correct metadata attributes including associated version after uploading a package.  I'll try to fix it in the next update, whenever that is, but it might take a few attempts.

**Comment by nirgilboa** [comment]
Sounds good - thanks!
